ext/pdo/pdo_sql_parser.re

```diff
@@ -70,8 +70,7 @@ static int scan(Scanner *s)
 struct placeholder {
 	const char *pos;
 	size_t len;
-	size_t qlen;	/* quoted length of value */
-	char *quoted;	/* quoted value */
+	zend_string *quoted;	/* quoted value */
 	int freeq;
```

Comment:
Do you need freeq still? You should be able to say that it should always be released (releasing interned strings does nothing).

Reply:
Wasn't sure in regards to the behaviour with interned strings, but will do this.